<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>valikko</class>
 <widget class="QDialog" name="valikko">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> *
{
font-family: century gothic;
color:white;
}
QPushButton
{
background:#e9de72;
border-radius:60px;

}
QPushButton
{
color:;
border-radius:15px;
}
QFrame
{
background:#b5aa47;
border-radius:15px;
}
</string>
  </property>
  <widget class="QPushButton" name="btnVaihdaTili">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Vaihda tili</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTilinro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ö</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTilityyppi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ö</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNostaRahaa">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta rahaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelaaTapahtumia">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selaa tapahtumia</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTervetuloa">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ö</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTalletaRahaa">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Talleta rahaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNaytaSaldo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Näytä saldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTilisiirto">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tilisiirto</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>871</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kirjaudu ulos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>btnVaihdaTili</zorder>
  <zorder>labelTilinro</zorder>
  <zorder>labelTilityyppi</zorder>
  <zorder>btnNostaRahaa</zorder>
  <zorder>btnSelaaTapahtumia</zorder>
  <zorder>labelTervetuloa</zorder>
  <zorder>btnTalletaRahaa</zorder>
  <zorder>btnNaytaSaldo</zorder>
  <zorder>btnTilisiirto</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
